packages/cli/src/utils/telemetry.ts

```diff
@@ -54,6 +55,12 @@ export async function sendTelemetry({
     if (!argv) {
       return;
     }
+
+    const hasInternet = await hasInternetConnectivity(1000);
```

Comment:
Can you name somehow `1000` or add comment?

Reply:
Maybe rename the function so the argument's meaning become obvious? Something like that?

```suggestion
    const hasInternet = await respondWithinMs(1000);
```